### PR TITLE
Meeting Notes

### PR DESCRIPTION
**Pull request by nabramow**

### CODE REVIEW

File: docs/meetings/dev/2024-12-17.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - A epic is something that needs all pieces done to be released to prod and can be done in a reasonable time, say 1-2 months.
+    - A sub issue is a thematic piece that encapsulates a specific part of the functionality for that epic, i.e. creating backend api, sharing event, making event page.
+    - In general we will stick with single tickets, but when a feature has many moving parts, the new subissue tickets are good to keep track of everything in one place.
+* Jesse: Right now only users can be deleted, but other things cannot. For moderation, it's important.
+    - Need api to delete things. Specifically comments and discussions.
+
+# Updates
+- Aapeli: Fixed the strong verification, merged UsersList component, some email fixes. Next: Discussions - notifications, admins ability to change discussion content. Badges PR needs review.
+- David: Took over Marco's PR and fixing the empty square on the map.
+- Jesse: Created a ticket to fix broken help center link, triage bug reports. Volume of support tickets increased this week, testing PRs. Next: Co
```
Comment (aapeliv) [inline docs/meetings/dev/2024-12-17.md]: What comes next!

File: docs/meetings/dev/2024-12-17.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Weekly meeting
+
+When: 12/24/24, 21:00 UTC.
```
Comment (aapeliv) [inline docs/meetings/dev/2024-12-17.md]: 17/12/24? Sorry for wrong side of pond convention

**Review by aapeliv — CHANGES_REQUESTED** [review]

File: docs/meetings/dev/2024-12-17.md (hunk 2 above)
Comment (nabramow) [reply docs/meetings/dev/2024-12-17.md]: lol I don't know what day it is thanks!

File: docs/meetings/dev/2024-12-17.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Present: Aapeli, David, Jesse, Nicole, Yannic, Gracee, Darren
+Scribe: Nicole
+
+# Notes
+
+* Aapeli talks about board meeting take-aways.
+    - Nicole joined the board.
+    - We're going to focus our identity on being a safe an active community.
+    - Ban on nudists
```
Comment (darrenvong) [inline docs/meetings/dev/2024-12-17.md]: Are we starting to see this a lot on our platform too? 😅

Comment (aapeliv) [reply docs/meetings/dev/2024-12-17.md]: Somewhat, can elaborate on Slack 😅

File: docs/meetings/dev/2024-12-17.md (hunk 1 above)
Comment (nabramow) [reply docs/meetings/dev/2024-12-17.md]: Ahaha whoops! Fixed!

**Review by aapeliv — APPROVED** [review]
LGTM!

**Pull request merged**